adapters/logicad/logicadtest/exemplary/banner.json

```diff
@@ -0,0 +1,92 @@
+{
+  "mockBidRequest": {
+    "id": "testid",
+    "imp": [
+      {
+        "id": "testimpid",
+        "banner": {
+          "format": [
+            {
+              "w": 300,
+              "h": 250
+            },
+            {
+              "w": 320,
+              "h": 50
+            }
+          ]
+        },
+        "ext": {
+          "bidder": {
+            "tid": "testtid"
+          }
+        }
+      }
+    ]
+  },
+  "httpCalls": [
+    {
+      "expectedRequest": {
+        "uri": "https://pbs.ladsp.com/adrequest/prebidserver",
```

Comment:
Consider using an obviously fake url for the tests, so there's no pressure to update in the future if the real endpoint changes.